packages/amplify-console-hosting/constants/question-constants.js

```diff
@@ -0,0 +1,19 @@
+import chalk from 'chalk';
+
+export const DEPLOY_TYPE_QUESTION = `Choose a ${chalk.red('type')}`;
+export const DEPLOY_TYPE_QUESTION_MANUAL = 'Manual deployment';
+export const DEPLOY_TYPE_QUESTION_CICD = 'Continuous deployment (Git-based deployments)';
+export const LEARN_MORE = 'Learn more';
+
+export const CICD_CONFIRM_QUESTION = 'Continuous deployment is configured in the Amplify Console. Please hit enter once you connect your repository';
+
+export const VIEW_APP_QUESTION = `You have set up CI/CD with Amplify Console. \
+Run ${chalk.green('git push')} from a connected branch to publish updates. \
+Open your Amplify Console app to view connected branches?`;
+
+export const APP_CONFIGURE_QUESTION = `We recommends you open ${chalk.green('AWS Amplify Console')} to configure advanced settings such as ${chalk.yellow('rewrite/redirect, basic auth, custom domain...')} Open your Amplify Console app to configure?`;
+export const APP_CICD_SERVE_QUESTION = 'You have set up Continuous deployment with Amplify Console. Visit Amplify Console to view your frontend URL?';
```

Comment:
You have set up continuous deployment with Amplify Console. Open your Amplify Console app to view connected branches?